### ISSUE TITLE
Search Page: Add missing title/lable for screenreader

### ISSUE THREAD

**Issue by geofranzi** [BEXIS2/Core#1910]
- [x] add
- [x] change placeholder text from "Search" to "Search within selected category"
- [x] facets are done here: https://github.com/BEXIS2/bexis2-core-ui/tree/1837_accessibility

![image](https://github.com/user-attachments/assets/f8afbb1b-4127-450b-9eae-0179c27f2f4d)

**Comment by geofranzi** [comment]
- [ ] rename "Filter categories" to "Search within a category"

**Comment by geofranzi** [comment]
label not so easy, filter cats als fine
